azure-pipelines.yml

```diff
@@ -92,6 +92,7 @@ jobs:
       arguments: '--blame --no-build -c $(buildConfiguration) -maxcpucount:1 /p:CopyLocalLockFileAssemblies=true $(skipFilter) --collect:"XPlat Code Coverage" --settings coverlet.runsettings --logger trx --results-directory $(Build.SourcesDirectory)'
       publishTestResults: false
   - task: PublishTestResults@2
+    condition: always()
```

Comment:
Did you mean to change this here?

Reply:
yeah, I had a random hystrix test fail, but because `dotnet test` failed this step didn't run and that failure wasn't published to the build results...  Previously the test step always auto-published the results so we didn't even have a discrete step here, but now we need to make sure it always runs so the results always show up in the report